oaknational/Oak-Web-Application
# Changed 'year group' and 'subject' order in unit modal

## Description
Added `<BulletList/>` into `<UnitModal/>` and changed 'year group' and 'subject'

## Issue(s)

Fixes `CUR-970`

## How to test

1. Go to https://deploy-preview-2896--oak-web-application.netlify.thenational.academy
2. Open a unit in the modal
3. Verify 'year group' and 'subject' are in the correct order
4. Test with PE Swimming units (Primary) and GCSE Computer Science for good measure
